Lib/test/test_dis.py

```diff
@@ -1335,6 +1335,32 @@ def test_call_specialize(self):
         got = self.get_disassembly(co, adaptive=True)
         self.do_disassembly_compare(got, call_quicken)
 
+    @cpython_only
```

Comment:
Can we move this test to a more appropriate location, like `test_opcache`?

Also, can we avoid hardcoding sequences of instructions into the tests and use a behavioral approach: testing that the behavior is the same with and without specialization.

We should already have tests for specialization of `COMPARE_OP` somewhere.

Reply:
Behavioural tests specifically for specialization sound like a great idea regardless of whether we're free-threaded or not. They should still make sure that actual specialization happened, and not just exercise some code. As far as I can tell we don't have any of those right now. I assume that was on purpose.

I agree we should already have tests for specialization of COMPARE_OP, but we don't. That's why I added the new test, similar to the existing test for specialization of BINARY_OP, which Matt adapted for free-threaded specialization in #123926. That test existed here, in test_dis. I assume they were added here on purpose. Should all those tests be moved to a more appropriate location?

Reply:
See:  https://github.com/python/cpython/pull/126498 I am not sure Mark intended this kinds of test.

Reply:
The test was replaced with a new test in test_opcache, along with the rest.